lib/chef_zero/cookbook_data.rb

```diff
@@ -31,15 +31,15 @@ def self.metadata_from(directory, name, version, recipe_names)
       # If both .rb and .json exist, read .rb
       # TODO if recipes has 3 recipes in it, and the Ruby/JSON has only one, should
       # the resulting recipe list have 1, or 3-4 recipes in it?
-      if has_child(directory, 'metadata.rb')
+      if has_child(directory, 'metadata.json')
+        metadata.from_json(File.join(directory.file_path, 'metadata.json'))
```

Comment:
I fixed this on my local install by using the filename function that's used below. Does it make sense to use that function here as well? Or is File.join safer, etc.?